pkg/localmetrics/localmetrics.go

```diff
@@ -16,6 +16,12 @@ var (
 		Help:      "Failed to obtain a valid pull secret",
 	}, []string{nameLabel})
 
+	MetricFleetSecretAbsent = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: metricsTag,
+		Name:      "fleet_secret_absent",
+		Help:      "Failed to obtain a vault secret for OCM service account",
+	}, []string{nameLabel})
+
```

Comment:
@weherdh - I missed this bit but we could remove the new metrics? We won't need it anymore based on our discussion.

Reply:
Regarding the expiry of the token, we don't check it yet but can look into it. Maybe a parallel thing to look into but isn't a blocker for immediate future. Regarding the removal of the metric itself, there was a discussion around leveraging existing metrics itself (when OAO connects to OCM) and use that instead of adding new metrics.